### PR TITLE
Feature/error404

### PR DESCRIPTION
**Pull request by cvalenzu**
- Added custom 404 error page
image
Can be customized with multiple images from the taxonomy and text.

### CODE REVIEW

File: layouts/error.vue [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
     <h1 v-else>
       {{ otherError }}
     </h1>
-    <NuxtLink to="/">
-      Home page
-    </NuxtLink>
   </v-app>
 </template>
 
 <script>
```
Comment (dirodriguezm) [inline layouts/error.vue:12]: We should be using nuxt-property-decorator syntax in all components to keep consistency.

File: layouts/error.vue [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+import Error404 from './errors/404.vue'
```
Comment (dirodriguezm) [inline layouts/error.vue:13]: Importing components is not needed. Just use them in template

File: layouts/error.vue [hunk 3]
```diff
@@ -23,7 +24,6 @@ export default {
   },
   data() {
```
Comment (dirodriguezm) [inline layouts/error.vue:25]: We should be using nuxt-property-decorator syntax to keep consistency

File: layouts/error.vue (hunk 2 above)
Comment (cvalenzu) [reply layouts/error.vue]: Fixed on ad45b59e259609c193f3ed27044db1eb94e93552

File: layouts/error.vue (hunk 3 above)
Comment (cvalenzu) [reply layouts/error.vue]: Fixed on ad45b59e259609c193f3ed27044db1eb94e93552

**Comment by cvalenzu** [comment]
- Added catch exception on object 404 (ad45b59e259609c193f3ed27044db1eb94e93552)

**Pull request merged**